<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AstrolabeMissionView</class>
 <widget class="QWidget" name="AstrolabeMissionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AstrolabeMissionView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="mAddEventButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mTransformEventButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mMoveEventUpButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mMoveEventDownButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="mRemoveEventButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="wsfg::spaceg::OrbitalMissionView" name="mMissionView">
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wsfg::spaceg::OrbitalMissionView</class>
   <extends>QTreeView</extends>
   <header>OrbitalMissionView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mAddEventButton</tabstop>
  <tabstop>mTransformEventButton</tabstop>
  <tabstop>mMoveEventUpButton</tabstop>
  <tabstop>mMoveEventDownButton</tabstop>
  <tabstop>mRemoveEventButton</tabstop>
  <tabstop>mMissionView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
